modules/nix-bitcoin.nix

```diff
@@ -42,6 +42,7 @@ in {
     ./liquid.nix
     ./spark-wallet.nix
     ./electrs.nix
+    ./onion-manager.nix
```

Comment:
You missed the historic opportunity to call this onion-chef, onion-dispenser, or onion-dealer! :)

Reply:
*almost missed ;P